src/main/java/com/goncalomb/bukkit/nbteditor/commands/CommandNBTItem.java

```diff
@@ -79,6 +79,27 @@ public boolean lore_addCommand(CommandSender sender, String[] args) throws MyCom
 		return true;
 	}
 
+	@Command(args = "lore set", type = CommandType.PLAYER_ONLY, minargs = 2, maxargs = Integer.MAX_VALUE, usage = "<index> <lore ...>")
+	public boolean lore_setCommand(CommandSender sender, String[] args) throws MyCommandException{
+		HandItemWrapper.Item item = new HandItemWrapper.Item((Player)sender);
+		List<String> loreList = item.meta.getLore();
+		if(loreList == null){
+			loreList = new ArrayList<String>();
+		}
+		Integer length = loreList.size();
+		Integer index = Utils.parseInt(args[0], -1);
+
```

Comment:
Needs test for negative index (see lore del)...

    if (index < 1) {
        sender.sendMessage("§cInvalid index. The index is an integer greater than 0.");
    } else

Also indexes start at 1: `loreList.set(index -1,`